test/e2e/density.go

```diff
@@ -181,6 +191,26 @@ var _ = Describe("Density", func() {
 			// Tune the threshold for allowed failures.
 			badEvents := BadEvents(events)
 			Expect(badEvents).NotTo(BeNumerically(">", int(math.Floor(0.01*float64(totalPods)))))
+
+			additionalNameStr := strconv.Itoa(minionCount) + "-" + string(util.NewUUID())
+			additionalRCName = "my-hostname-density" + additionalNameStr
+			Logf("Schedling additional Pods to measure startup latencies")
+			config = RCConfig{Client: c,
+				Image:         "gcr.io/google_containers/pause:go",
+				Name:          additionalRCName,
+				Namespace:     ns,
+				PollInterval:  itArg.interval,
+				PodStatusFile: fileHndl,
+				Replicas:      minionCount,
+			}
+			latencies, err := RunRC(config)
+			expectNoError(err)
+			sort.Float64s(latencies)
+			perc50 := latencies[len(latencies)/2]
+			perc90 := latencies[(len(latencies)*9)/10]
+			perc99 := latencies[(len(latencies)*99)/100]
+			Logf("Latencies: %v", latencies)
```

Comment:
Theres a log around line 175 in density.go:
INFO: E2E startup time for 300 pods: 2m23.150119007s

It does not print normalized throughput, but one can easily do it. For me it varies from 100 to 200  pods per minute. It counts whole `RunRC` call, so it includes all additional validation we have, creation of RC and so on.